Tools/clinic/libclinic/converters.py

```diff
@@ -1228,3 +1228,30 @@ def set_template_dict(self, template_dict: TemplateDict) -> None:
             type_object = cls.type_object
             type_ptr = f'PyTypeObject *base_tp = {type_object};'
             template_dict['base_type_ptr'] = type_ptr
+
+
+# Converters for var-positional parameter.
+
+class varpos_tuple_converter(CConverter):
+    type = 'PyObject *'
+    format_unit = ''
+    c_default = 'NULL'
+
+    def cleanup(self) -> str:
+        return f"""Py_XDECREF({self.parser_name});\n"""
+
+    def parse_arg(self, argname: str, displayname: str, *, limited_capi: bool) -> str | None:
+        raise AssertionError('should never be called')
+
+class varpos_array_converter(CConverter):
+    type = 'PyObject * const *'
+    format_unit = ''
+    length = True
+    c_ignored_default = ''
+
+    def parse_arg(self, argname: str, displayname: str, *, limited_capi: bool) -> str | None:
+        raise AssertionError('should never be called')
+
+# XXX: temporary
+class varpos_object_converter(varpos_tuple_converter):
+    pass
```

Comment:
```suggestion
```
```suggestion

# XXX: temporary
class varpos_object_converter(varpos_tuple_converter):
    pass
```

Do we need this alias?  grep shows that it's mostly used in tests.  I see two other examples, one is the print function.  I think using the array converter instead will offer some speedup.  Do you mind to do this in current pr?

Reply:
I will do this in the following PR. I do not want to make this PR even larger.

I do not understand what is in the proposed suggestion.